sourcegraph/sourcegraph
# lucky search: initial styling work

Initial styling work for lucky search based on the [Figma design](https://www.figma.com/file/GikUoxdCrRTmbQpiu72AUi/UX-for-%E2%80%99feeling-lucky%E2%80%99-search-mode.?node-id=2521%3A6607#252439547). This required a little bit of backend work to split up the description from the result count, but otherwise is mostly frontend work. This is not a final result, but a starting point based on the parts of the Figma design with the most clarity.

![image](https://user-images.githubusercontent.com/206864/186782562-e8d6018e-a43e-40a8-ad39-aceeb2505dbc.png)


## Test plan

- Verify new styling
- Update go tests base don backend changes
